core/storage/feedback/gae_models.py

```diff
@@ -73,9 +73,18 @@ class FeedbackThreadModel(base_models.BaseModel):
 
     @classmethod
     def generate_new_thread_id(cls, exploration_id):
-        """Generates a new thread id, unique within the exploration.
+        """Generates a new thread ID which is unique within the exploration.
 
-        Exploration ID + the generated thread ID is globally unique.
+        Args:
+            exploration_id: str, ID associated with each exploration
```

Comment:
Also, I suggest:

    exploration_id: str. The ID of the exploration.

(There's only one exploration here, so no need for "each".)